bin/upgrading/update-db-general.pl

```diff
@@ -1201,7 +1201,7 @@
     dmid    TINYINT UNSIGNED NOT NULL,
     itid    MEDIUMINT UNSIGNED AUTO_INCREMENT NOT NULL,
     PRIMARY KEY (dmid, itid),
-    itcode  VARCHAR(80) NOT NULL,
+    itcode  VARCHAR(120) CHARACTER SET ascii NOT NULL,
```

Comment:
I'm actually unfamiliar with what this particular syntax does. What behavior does this impact for the DB?